src/style.css

```diff
@@ -0,0 +1,39 @@
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+.stars {
+  display: flex;
+}
+
+.stars__star {
+  background: url("images/star.svg") no-repeat center;
+  width: 16px;
+  height: 16px;
+}
+
+.stars__star:not(:last-child) {
+  margin-right: 4px;
+}
+
+.stars--1 .stars__star:nth-child(-n+1) {
+  background: url("images/star-active.svg") no-repeat center;
+}
+
+.stars--2 .stars__star:nth-child(-n+2) {
+  background: url("images/star-active.svg") no-repeat center;
+}
+
+.stars--3 .stars__star:nth-child(-n+3) {
+  background: url("images/star-active.svg") no-repeat center;
+}
+
+.stars--4 .stars__star:nth-child(-n+4) {
+  background: url("images/star-active.svg") no-repeat center;
+}
+
+.stars--5 .stars__star:nth-child(-n+5) {
+  background: url("images/star-active.svg") no-repeat center;
+}
```

Comment:
combine all these selectors , to prevent code duplicating

Reply:
done!

Reply:
> combine all these selectors , to prevent code duplicating

ready